src/main/java/com/volasoftware/tinder/controllers/FriendSeederController.java

```diff
@@ -37,27 +36,25 @@ public class FriendSeederController {
             @ApiResponse(code = 400, message = "Invalid ID provided!"),
             @ApiResponse(code = 404, message = "Resource not found!")
         })
-    @GetMapping
+    @GetMapping("")
     public ResponseEntity<?> seedFriends(
-        @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
         @Valid
         @Min(0L)
-        @RequestParam(name = "id")
-        Optional<Long> accountId,
+        @RequestParam(required = false) Long accountId,
         @RequestParam(value = "page", defaultValue = "0") int page,
         @RequestParam(value = "size", defaultValue = "10") int size) {
-        
+
         Pageable pageable = PageRequest.of(page, size);
-        if (accountId.isEmpty()) {
+        if (accountId != null) {
```

Comment:
This logic can be moved in the service class and internally to handle calling the proper method (`linkRequestedAccountWithBots()` or `linkAllAccountsWithBots()` )